### PR TITLE
fix(common): avoid crashes when no name/title and required prop given

### PR DESCRIPTION
**Pull request by Befkadu1**
When an object has a title or a name, the function returns the title or a name. If no title or
name given, it will return the required property value as a title. In our case the property is
neither a title/name nor a required property which caused the crash. This PR solves the issue

fix Lundalogik/crm-feature#2857

## Review:
- [x] Commits are [atomic](https://seesparkbox.com/foundry/atomic_commits_with_git)
- [x] Commits have the correct *type* for the changes made
- [x] Commits with *breaking changes* are marked as such

### Browsers tested:
(Check any that applies, it's ok to leave boxes unchecked if testing something didn't seem relevant.)

Windows:
- [ ] Chrome
- [ ] Edge
- [ ] Firefox

Linux:
- [ ] Chrome
- [ ] Firefox

macOS:
- [x] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [ ] iOS

### CODE REVIEW

File: src/components/form/templates/common.spec.ts [hunk 1]
```diff
@@ -183,9 +184,13 @@ describe('findTitle()', () => {
...
             output: 'test',
         },
+        {
+            input: [{ props: { limetype: 'deal' } }, schema, schema],
+            output: null,
+        },
     ].forEach(({ input, output }) => {
         const inputJson = JSON.stringify(input[1]);
-        it(`returns ${output} when called with ${inputJson}`, () => {
+        it.only(`returns ${output} when called with ${inputJson}`, () => {
```
Comment (adrianschmidt) [inline src/components/form/templates/common.spec.ts:193]:
```suggestion
        it(`returns ${output} when called with ${inputJson}`, () => {
```

Comment (Befkadu1) [reply src/components/form/templates/common.spec.ts:193]: omg

Comment (Befkadu1) [reply src/components/form/templates/common.spec.ts]: ⚡

File: src/components/form/templates/common.ts [hunk 2]
```diff
@@ -56,10 +56,15 @@ export function findTitle(data: any, fieldSchema: any, formSchema: any) {
     data = sortDataByProperties(data, subSchema.properties);
 
     const firstEntry = findFirstEntry(data, subSchema);
+
```
Comment (adrianschmidt) [inline src/components/form/templates/common.ts]: I removed this. I think it's unnecessary, but if you really want it, the change belongs in a separate commit 🙂

**Review by adrianschmidt — APPROVED** [review]
